frontend/src/pages/Register.jsx

```diff
@@ -1,7 +1,206 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import Typography from "@mui/material/Typography";
+import { Grid } from "@mui/material";
+import axios from "axios";
+import { toast } from "react-toastify";
+import Input from "../components/Input";
+
 
 const Register = () => {
-  return <div>Register</div>;
+  const [customers, setCustomers] = useState([]);
+  const [customer, setCustomer] = useState({
+    firstname: "",
+    lastname: "",
+    driverLicense: "",
+    email: "",
+    password: "",
+    phoneNumber: "",
+    dob: "",
+    goldMembership: false,
+    province: "",
+    city: "",
+    postalCode: "",
+    streetNumber: "",
+    streetName: "",
+    unitNumber: "",
+  });
+
+  const handleChange = (e) => {
+    setCustomer({ ...customer, [e.target.name]: e.target.value });
+  };
+  const getCustomers = () => {
+    axios.get(`${process.env.REACT_APP_SERVER_URL}/customer/`).then((res) => {
+      console.log(res.data);
+      setCustomers(res.data);
+    });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log(customer);
+    axios
+      .post(`${process.env.REACT_APP_SERVER_URL}register/`, {
+        ...customer,
+      })
+      .then((res) => {
+        if (res.status === 201) {
+          setCustomers((prevCustomers) => [...prevCustomers, { ...customer }]);
```

Comment:
you can delete this line, instead here you need to use the same logic as the login, set the local storage info and navigate to the main customer page instead